packages/orchestrator-ui-components/src/components/WfoForms/formFields/SplitPrefix.tsx

```diff
@@ -98,15 +98,22 @@ export default class SplitPrefix extends React.PureComponent<IProps> {
     };
 
     render() {
+        // IPv4 subnet size should be between /32 and /12
+        // IPv6 subnet size should be between /128 and /32
         const { id, name, subnet, prefixlen, prefixMin, selectedSubnet } =
             this.props;
         const version = subnet.indexOf(':') === -1 ? 4 : 6;
-        const max_for_version = version === 4 ? 32 : prefixMin + 11;
+        const minPrefixLength = version === 4 ? 12 : 32;
```

Comment:
This logic is a bit of a black box for me. prefixMin is not considered anymore and that is passed in by the backend I think. Is it certain no logic depends on that?

Reply:
Agree, it is bit confusing, but I tested it thouroughly.

`prefixMin` is used a bit further down in the code, but not used if the minPrefixLength is larger

```
        const minimalSelectablePrefixLength =
            minPrefixLength > prefixMin ? minPrefixLength : prefixMin;
```

Reply:
We might want to move such logic to a util file and add unit tests.

Reply:
Good suggestion. However it is not critical (used only in this form field) code, and there is a manual workaround available in the form.